### PR TITLE
Add link to packaging helpers

### PR DESCRIPTION
**Pull request by decentral1se**
As recommended by @kay0u.

Rendered version:

> https://github.com/YunoHost/example_ynh/blob/3046ec3fdc9ddcbf198cdae8d08c7f2980c32cc4/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - Edit `conf/nginx.conf` file to match application prerequisites.
 - Edit `manifest.json` with application specific information.
 - Edit the `install`, `upgrade`, `remove`, `backup`, and `restore` scripts.
+  - Using the [script helpers documentation](https://yunohost.org/#/packaging_apps_helpers)
```
Comment (JimboJoe) [inline README.md]: On my French browser, the page doesn't exist... whereas http://helpers.yunohost.org/ points to the English page (no French page yet). Shall we use that link instead (is it official)?

Comment (kay0u) [reply README.md]: That's weird... But if it's working

Comment (decentral1se) [reply README.md]: Uhhhh, should I use this other new URL?

Comment (decentral1se) [reply README.md]: OK, fixed in https://github.com/YunoHost/example_ynh/pull/94/commits/4327e037eec95a0732cc94edd370985d934747b3. Points to the helpers ... URL.

**Comment by decentral1se** [comment]
OK, this should be good to go then.

**Pull request merged**